kinesify-data.js

```diff
@@ -0,0 +1,100 @@
+// /**
```

Comment:
Just add this file to .gitignore if it's just a local helper thing.

Reply:
Done!